pennylane/transforms/core/transform.py

```diff
@@ -50,7 +51,7 @@ def transform(
             ``quantum_transform``.
         classical_cotransform (Callable): A classical co-transform is a function to post-process the classical
             jacobian and the quantum jacobian and has the signature: ``my_cotransform(qjac, cjac, tape) -> tensor_like``
-        is_informative (bool): Whether or not a transform is informative. If true the transform is queued at the end
+        is_informative=False (bool): Whether or not a transform is informative. If true the transform is queued at the end
```

Comment:
Do we normally do this for defaults in the docstrings?

Reply:
We could potentially move the option keyword arguments to a "Keyword Args" section instead of using the "Args" section.

Reply:
I'm still a bit unsure here on what is special about `is_informative` for us to add `=False`

Reply:
I'm separating it out into a "Arrgs" and "Keyword Args" section to include the default values for the Keyword Args.

Reply:
Basically, I didn't realize that the default value originally was `None` and it caused me a headache. Hence why I want to be a bit more explicit about what the default values are.